packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -67,21 +67,6 @@ export class AnchorMenuItem extends AnchorBase {
         return true;
     }
 
-    // The FAST Menu manages the `tabindex` of its menu items. Because of a bug in Chromium
-    // (https://bugs.chromium.org/p/chromium/issues/detail?id=1346606), setting the tabindex on an element
-    // with `delegatesFocus=true` causes the element to lose focus. This causes the menu to close, preventing
-    // arrowing through the items or navigating to the url. As a workaround, we intercept attempts to
-    // set the tabindex on the host and instead set it on the inner anchor.
-    // The referenced Chromium bug is actually fixed, but it hasn't been pulled into Edge yet (it is in
-    // Chrome). Issue https://github.com/ni/nimble/issues/1118 tracks removal of this workaround.
-    public override setAttribute(qualifiedName: string, value: string): void {
-        if (qualifiedName === 'tabindex') {
-            this.anchor.setAttribute(qualifiedName, value);
-        } else {
-            super.setAttribute(qualifiedName, value);
-        }
-    }
-
     // This is part of the bug workaround described above (in setAttribute)
```

Comment:
Did you scan this file? There are two comments below saying the source is related to this workaround?

Reply:
Good catch. I had just searched for the issue number, so I didn't notice the getter/setter.